ttnn/cpp/ttnn/operations/eltwise/ternary/CMakeLists.txt

```
@@ -3,9 +3,26 @@ add_library(TTNN::Ops::Eltwise::Ternary ALIAS ttnn_op_eltwise_ternary)
 
 target_precompile_headers(ttnn_op_eltwise_ternary REUSE_FROM TT::CommonPCH)
 TT_ENABLE_UNITY_BUILD(ttnn_op_eltwise_ternary)
+set_target_properties(
+    ttnn_op_eltwise_ternary
+    PROPERTIES
+        VERIFY_INTERFACE_HEADER_SETS
+            FALSE
```

Looks like this one has host-side APIs.  We should put those through the gauntlet.  To do that while omitting the device-side kernels from trying to be processed on the host platform (recipe for sadness) do this:

```suggestion
        INTERFACE_HEADER_SETS_TO_VERIFY
            api
```